config/releases.exs

```diff
@@ -59,8 +59,8 @@ config :service_gather, Gather.Application,
     ],
     handlers: [Gather.Event.Handler],
     storage: [
-      module: Brook.Storage.Ets,
-      init_arg: []
+      module: Brook.Storage.Redis,
+      init_arg: [redix_args: [host: "localhost"], namespace: "service:gather:view"]
```

Comment:
since the `releases.exs` file is intended to capture prod-ready settings, i'd suggest pulling all of these `[host: "localhost"]` out into a single variable near the top of the file (see how we reference `kafka_endpoints` and then define it as `System.get_env("REDIS_HOST") || [host: "localhost"]` to pull the value from the helm charts if they're supplied.

Reply:
I just changed these to use `System.get_env("REDIS_HOST")`. Should I add `localhost` as a backup still?

Reply:
as of elixir 1.9 it's considered best practice to include a failover value for `System.get_env` calls, so i would.